halotools/empirical_models/abunmatch/tests/test_bin_free_cam.py

```diff
@@ -384,11 +386,13 @@ def test_subgrid_noise1():
     result2 = conditional_abunmatch(x, y, x2, y2, nwin1, add_subgrid_noise=True)
     assert np.allclose(result, result2, atol=0.1)
     assert not np.allclose(result, result2, atol=0.02)
+    assert np.all(result - result2 != 0)
```

Comment:
This should be enough to assert that we are adding noise to the leftmost/rightmost edges.

Reply:
Does this actually ensure that noise was added to the edges? Or just that noise was added *somewhere*?

Reply:
I think it ensures that noise was added to every point